src/services/auth-service.ts

```diff
@@ -0,0 +1,44 @@
+import { initializeApp } from 'firebase/app';
+import { Auth, getAuth, signInWithEmailAndPassword } from "firebase/auth";
+import { getEnvVar } from "../helpers/envVars";
+import userProps from "../types/UserProps";
+
+export class AuthService {
+    private static apiKey?: string
+    private static authDomain?: string
+    private static databaseURL?: string
+    static projectId?: string
+    private static storageBucket?: string
+    private static messagingSenderId?: string
+    private static appId?: string
+    private static auth: Auth
+
+    public static initialize(): void {
+        AuthService.apiKey = getEnvVar('FIREBASE_API_KEY')
+        AuthService.authDomain = getEnvVar('FIREBASE_AUTH_DOMAIN')
+        AuthService.databaseURL = getEnvVar('FIREBASE_DATABASE_URL')
+        AuthService.projectId = getEnvVar('FIREBASE_PROJECT_ID')
+        AuthService.storageBucket = getEnvVar('FIREBASE_STORAGE_BUCKET')
+        AuthService.messagingSenderId = getEnvVar('FIREBASE_MESSAGING_SENDER_ID')
+        AuthService.appId = getEnvVar('FIREBASE_APP_ID')
+
+        initializeApp({
+            apiKey: AuthService.apiKey,
+            authDomain: AuthService.authDomain,
+            databaseURL: AuthService.databaseURL,
+            projectId: AuthService.projectId,
+            storageBucket: AuthService.storageBucket,
+            messagingSenderId: AuthService.messagingSenderId,
+            appId: AuthService.appId,
+        })
+
+        AuthService.auth = getAuth();
+    }
+
+    public static signIn(user: userProps) {
+        if(!user.email){
+            throw new Error("Please, provide a valid email")
+        }
+        return signInWithEmailAndPassword(AuthService.auth, user.email, user.password)
```

Comment:
We could check here first if the AuthService indeed has the `auth` initialized, but since we are doing this in a hurry it's fine by now

Reply:
Mmmm, I have to give it a thought because auth is going to be used in all the methods of AuthService (in other branches that I am working on) and I don't want to repeat that check for the initialisation everytime, but I'll think about it